### PR TITLE
Fixed #4975 Deployment Production system

### LINKED ISSUE #4975: Feedback on page: /guides/v2.3/config-guide/deployment/pipeline/
# General issue

## Description:

Describing 'production system' it says 'no changes to websites, stores, store views, products, or the catalog. '.  This implies that you wouldn't update product data on the live system and would somehow deploy it. I believe the intent is 'no changes to product view configuration' or similar.

## Possible solutions:

Replace 'products' with 'product view settings'.

### PR DESCRIPTION
**Pull request by kirtinariya1**
## Purpose of this pull request

This pull request (PR) will be merge the show correct content in the Deployment Overview of Production system

## Affected DevDocs pages
- https://devdocs.magento.com/guides/v2.2/config-guide/deployment/pipeline/
- https://devdocs.magento.com/guides/v2.3/config-guide/deployment/pipeline/

## Links to Magento source code
N/A

### CODE REVIEW

**Comment by magento-cicd2** [comment]
[CLA assistant check](https://cla.dev.magento.com/magento/devdocs?pullRequest=5017) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla.dev.magento.com/magento/devdocs?pullRequest=5017) before we can accept your contribution.<br/>**1** out of **2** committers have signed the CLA.<br/><br/>:white_check_mark: kirtinariya1<br/>:x: Kirti Nariya<hr/>**Kirti Nariya** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: _includes/config/split-deploy/split-deploy-overview.md [hunk 1]
```diff
@@ -22,4 +22,4 @@ Staging system
 :	_Optional_. You can optionally set up a staging system to use for final testing of all integrated code, including User Acceptance Testing (UAT). Set up a staging system the same way you set up a production system. Except for the fact that staging is not your live store and doesn't process orders from customers, it's identical to production.
 
 Production system
-:	Your live store. You should make minimal configuration changes here and no changes to websites, stores, store views, products, or the catalog. You should make all those types of changes in your development system.
+:	Your live store. You should make minimal configuration changes here and no changes to websites, stores, store views, product view settings, or the catalog. You should make all those types of changes in your development system.
```
Comment (osrecio) [inline _includes/config/split-deploy/split-deploy-overview.md]: Why only in product view settings and not products in general. 

We can say the same of category view settings?

Comment (kirtinariya1) [reply _includes/config/split-deploy/split-deploy-overview.md]: @osrecio Thank for information. I appreciate for that i have updated to
"Your live store. You should make minimal configuration changes here and no changes to websites, stores, store views, products, product view settings, or the catalog, categories, category view settings. You should make all those types of changes in your development system."

Comment (kirtinariya1) [reply _includes/config/split-deploy/split-deploy-overview.md]: @osrecio Changes has done.

**Pull request merged**